src/ui/public/doc_table/doc_table.html

```diff
@@ -104,6 +104,11 @@
   <kbn-infinite-scroll ng-if="infiniteScroll" more="addRows"></kbn-infinite-scroll>
 </div>
 
+<div ng-if="!columns.length" class="table-vis-error">
+  <h2><i class="fa fa-meh-o"></i></h2>
```

Comment:
I think it would be good to clean this up while we're touching it. At least that's what I try to do as long as the changes aren't too invasive. So we would change this to:

```html
<div ng-if="!columns.length" class="table-vis-error">
  <p class="kuiSubTitle kuiVerticalRhythm">
    <span class="kuiIcon fa-meh-o"></span>
  </p>
  
  <p class="kuiText kuiVerticalRhythm">
    No columns selected. (or whatever you think is best here)
  </p>
</div>
```